File: src/main/java/OpenJvmOptionsDialogAction.java
import com.intellij.openapi.actionSystem.AnAction;
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.kubicz.mavenexecutor.window.MavenExecutorService;
import com.kubicz.mavenexecutor.window.MavenExecutorSetting;
import myToolWindow.JvmOptionsDialog;


public class OpenJvmOptionsDialogAction extends AnAction {

    public OpenJvmOptionsDialogAction() {
        super("");
    }

    public void actionPerformed(AnActionEvent event) {
        MavenExecutorSetting setting = MavenExecutorService.getInstance(event.getProject()).getSetting();

        JvmOptionsDialog jvmOptionsDialog = new JvmOptionsDialog(event.getProject());
        jvmOptionsDialog.setJvmOptions(setting.getJvmOptions());

        if(jvmOptionsDialog.showAndGet()) {
            setting.setJvmOptions(jvmOptionsDialog.getJvmOptions());
        }
    }

}

File: src/main/java/OpenJvmOptionsDialogAction.kt
import com.intellij.openapi.actionSystem.AnAction
import com.intellij.openapi.actionSystem.AnActionEvent
import org.kubicz.mavenexecutor.view.window.MavenExecutorService
import myToolWindow.JvmOptionsDialog


class OpenJvmOptionsDialogAction : AnAction("") {

    override fun actionPerformed(event: AnActionEvent) {
        val setting = MavenExecutorService.getInstance(event.project!!).currentSettings

        val jvmOptionsDialog = JvmOptionsDialog(event.project)
        jvmOptionsDialog.jvmOptions = setting.jvmOptions

        if (jvmOptionsDialog.showAndGet()) {
            setting.jvmOptions = jvmOptionsDialog.jvmOptions
        }
    }

}